src/vmm/src/devices/acpi/vmgenid.rs

```diff
@@ -148,11 +138,8 @@ impl<'a> Persist<'a> for VmGenId {
         }
     }
 
-    fn restore(
-        constructor_args: Self::ConstructorArgs,
-        state: &Self::State,
-    ) -> Result<Self, Self::Error> {
-        Self::from_parts(GuestAddress(state.addr), state.gsi, constructor_args.mem)
+    fn restore(mem: Self::ConstructorArgs, state: &Self::State) -> Result<Self, Self::Error> {
+        Ok(Self::from_parts(GuestAddress(state.addr), state.gsi))
```

Comment:
`mem` is not used since now we do the guest memory write in `VmGenId::activate()`. Do we still need to take it as `mem`? (not sure if it's possible, but could make `ConstructorArgs = ()` and take it as `_` instead of `mem`?)

Reply:
Can we take the arg as `_`? like:
```suggestion
    fn restore(_: Self::ConstructorArgs, state: &Self::State) -> Result<Self, Self::Error> {
        Ok(Self::from_parts(GuestAddress(state.addr), state.gsi))
```

Reply:
done